zone/entity.h

```diff
@@ -60,6 +60,8 @@ class Bot;
 
 extern EntityList entity_list;
 
+constexpr std::string SEE_BUFFS_FLAG = "see_buffs_flag";
```

Comment:
Sorry, I misspoke, this should actually be `constexpr const char*`. I'd edit it myself, but you're on a different repository.